core/unix/loader.c

```diff
@@ -2311,7 +2347,7 @@ privload_early_inject(void **sp, byte *old_libdr_base, size_t old_libdr_size)
         interp_map = elf_loader_map_phdrs(
             &interp_ld, false /* fixed */, os_map_file, os_unmap_file, os_set_protection,
             privload_check_new_map_bounds, memset,
-            privload_map_flags(MODLOAD_IS_APP /*!reachable*/));
+            privload_map_flags(MODLOAD_IS_APP /*!reachable*/), NULL /*remap_func*/);
```

Comment:
Why is this NULL instead of the new func?

Reply:
Done. I initially didn't plan on handling other callers in this PR.